dateparser/languages/loader.py

```diff
@@ -34,10 +36,14 @@ def _load_data(self):
             data = get_data('data', 'languages.yaml')
         else:
             data = self.file.read()
-        data = load_yaml(data)
-        base_data = data.pop('base', {})
+        
+        if not self._raw_data:
```

Comment:
Does the following make sense? I am trying to force parsing of yaml file only once. Without this implementation, it will be parsed every time a new `LanguageDataLoader` instance is created.

```
   def _load_data(self):

        if not LanguageDataLoader._raw_data:
            if self.file is None:
                data = get_data('data', 'languages.yaml')
            else:
                data = self.file.read()
            LanguageDataLoader._raw_data = load_yaml(data)
```